tests/unit/button/button_options.js

```diff
@@ -6,152 +6,101 @@
 module( "button: options" );
 
 test( "disabled, explicit value", function( assert ) {
-	expect( 7 );
+	expect( 9 );
 
-	var element = $( "#radio01" ).button({ disabled: false });
+	var element = $( "#button" ).button({ disabled: false });
 	deepEqual( element.button( "option", "disabled" ), false, "disabled option set to false" );
-	deepEqual( element.prop( "disabled" ), false, "element is disabled" );
+	deepEqual( element.prop( "disabled" ), false, "Disabled property is false" );
```

Comment:
The message argument is redundant. The assertion should be `ok()`. `strictEqual()` would be fine as well.